content/docs/iac/concepts/resources/providers.md

```diff
@@ -50,7 +50,42 @@ For example, to generate a local SDK for the [`hashicorp/random` provider](https
 $ pulumi package add terraform-provider hashicorp/random
 ```
 
+{{% notes type="tip" %}}
 In order to make sure Pulumi users are aware of the Any Terraform Provider's capabilities, Pulumi has included select, popular providers that can be consumed in Pulumi via the Any Terraform Provider in the Pulumi Registry, such as [The Honeycomb provider](/registry/packages/honeycombio/).
+{{% /notes %}}
+
+The generated SDK will include a `.gitignore` so it can be safely committed to version control without including all of the SDK's dependencies. The SDK contains instructions to download the provider binary to a shared (and cached) location on your local system outside of the working directory - the binary is not committed to version control.
+
+#### Adding Provider Packages to the Project Configuration File
+
+{{% notes type="info" %}}
+Adding provider packages to your project configuration file requires Pulumi version 3.157.0 or later.
+{{% /notes %}}
+
+You can avoid the need to commit any generated SDK files to version control by adding your parameterized packages to your Pulumi project configuration file (`Pulumi.yaml`).
+
+You can add a list of your named provider packages under the [`packages`](/docs/iac/concepts/projects/project-file/#packages-options) key. For the example in the previous section, the syntax would be:
+
+```yaml
+packages:
+  random: # This is an arbitrary name
```

Comment:
Its only arbitrary for "non-local" providers. If your using "source based providers" i.e. where you've checked the provider source code in side-by-side with your project then this _must_ match the provider name.

Not sure there's a good way to put that in here but figured worth pointing out that detail.

Reply:
It's not arbitrary: `package add` will add it as `random` and that's what the user should use. We should not encourage arbitrary names here.